build.proj

```diff
@@ -33,6 +33,8 @@
     <ExeExtension>.exe</ExeExtension>
     <ExeExtension Condition=" '$(OS)' != 'Windows_NT' "></ExeExtension>
     <DotnetStage0>$(Stage0Path)/dotnet$(ExeExtension)</DotnetStage0>
+
+    <DotnetCliBuildDirectory>$(MSBuildThisFileDirectory)/build_projects/dotnet-cli-build/</DotnetCliBuildDirectory>
```

Comment:
(nit) - I know this was the way it was before, but I think this should use $(RepoRoot) instead of $(MsBuildThisFileDirectory), so it is more portable and readable.